sources/packages/backend/apps/api/src/auth/decorators/institution/institution-bc-public.decorator.ts

```diff
@@ -0,0 +1,6 @@
+import { SetMetadata } from "@nestjs/common";
+
+// Allow only BC public institution to access the API.
+export const INSTITUTION_IS_BC_PUBLIC_KEY = "institution-is-bc-public-key";
+export const IsBcPublicInstitution = () =>
```

Comment:
Even though we missing sometimes these decorators also deserve some comments because they are the ones applied to controllers.